kse/src/org/kse/gui/actions/resources.properties

```diff
@@ -244,7 +244,7 @@ ExportTrustedCertificatePublicKeyAction.text
 ExportTrustedCertificatePublicKeyAction.tooltip                                  = Export Trusted Certificate entry's public key
 
 FindAction.Find.Title       = Find
-FindAction.NotFound.message = Not found
+FindAction.NotFound.message = ''{0}'' Not found
```

Comment:
The n of "not" should be lower case now

Reply:
You are right.
_But the issue was also before my PR._
I will correct them.